Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -464,6 +464,22 @@
       Cable: 5
       Glass: 2
 
+- type: entity
+  id: SpaceHeaterMachineCircuitBoard
+  parent: BaseMachineCircuitboard
+  name: space heater machine board
+  description: A PCB for a space heater.
```

Comment:
Please expand the abbreviation "PCB". I can tell this was copy/pasted from scrubbers.

Reply:
> I can tell this was copy/pasted from scrubbers.
It was 😁  (fixing both)